loopchain/baseservice/slot_timer.py

```diff
@@ -54,12 +59,14 @@ def call(self):
         util.logger.spam(f"call slot({self.__slot}) delayed({self.__delayed})")
         if self.__slot > 0:
             self.__slot -= 1
-            self.__timer_service.get_event_loop().create_task(self.__callback())
+            asyncio.get_event_loop().create_task(self.__callback())
```

Comment:
this is not root cause. please see https://github.com/icon-project/loopchain/pull/168